web/src/components/custom_timeline/custom_timeline.js

```diff
@@ -0,0 +1,25 @@
+import { Box } from "@mui/material";
```

Comment:
Is there a reason we put this in its own component instead of under the classes / schedule directory?

Reply:
It might be useful in other places than just the class schedule, so I made it available at a higher level

Reply:
timeline now part of schedule